scripts/setup/setup.bat

```diff
@@ -27,19 +27,19 @@ IF ERRORLEVEL 1 (
 where /q node
 IF ERRORLEVEL 1 (
 	ECHO Error: node is not installed.
-	ECHO Please Install NodeJS from: https://nodejs.org/en/download/
+	ECHO Please Install NodeJS from: https://nodejs.org/en/download
 	ECHO And make sure its in the path
     GOTO :end
 )
 
-where /q npm
+where /q pnpm

```

Comment:
you are only checking if pnpm is installed, please allow both npm OR pnpm

Reply:
I have fixed it.